gtp2ogs.js

```diff
@@ -19,7 +20,7 @@ process.on('uncaughtException', function (er) {
   console.trace("ERROR: Uncaught exception");
   console.error(`ERROR: ${er.stack}`);
   if (!conn || !conn.socket) {
-    conn = new Connection();
+    conn = new Connection(io, config);
```

Comment:
we'd need to test this on beta ogs to make sure it works as intended in real gtp2ogs, not just testing

Reply:
You dont trust my amazing changes? 😨

Reply:
you know what i need to trust you, is more amazing changes
looooooooooooooooooooool
(kidding, i'll quickly test them on beta lol while also testing mine, keep sending more i so love it !!!! :100: :100: :100: :100: :100: )

Reply:
Ha I'm kidding. This part is not even tested so of course we need to do a quick sanity check ourself :)

Reply:
silly dorus!! :scold:!!! :smile: